template/navbar.html

```diff
@@ -1,5 +1,17 @@
 {{ define "navbar" }}
 
-<h1>Navigation Bar</h1>
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="#">Score Board</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item active">
+          <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
```

Comment:
Whitespace after "Home"

Reply:
idk :), this is copy pasted from https://getbootstrap.com/docs/4.0/components/navbar/#nav
i can fix it if you insist :)

Reply:
done